nengo_sphinx_theme/theme/sidebar.html

```diff
@@ -25,3 +25,6 @@
 
   </ul>
 </div>
+{% if building_version %}
```

Comment:
I think we'll want this inside the div, so that it stays in place relative to the sidebar (with the sticky sidebar fix).  (Minor change I can do myself, I'm just writing it down here so I don't forget)

Reply:
It's technically faster not to include the template at all, but we're talking one or two milliseconds probably so whoever touches the file next can take it out